### DISCUSSION TITLE
Favorites List in Lists Plugin

### DISCUSSION THREAD

**Discussion by SouthHamptonSTL** [Stellarium/stellarium#2285] [Ideas]
I would like to be able to have a personal list to store my favorite sky objects for easy access to items I like.  I would like to be able to add object to the list via type and save or right click and "Add to Favorites" from any of the search lists.

**Comment by gzotti** [comment]
If you want a feature added, please specify a "feature wish" Issue. The "Discussions" are rather for discussions about existing features. 

This said, have you tried the observing lists? They are work in progress but (at least mostly) fulfill your wishes.
